net-analyzer/cottontail/cottontail-20190426.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/QKaiser/cottontail.git"
+	KEYWORDS=""
+else
+	KEYWORDS="~amd64 ~x86"
+	EGIT_COMMIT="77ed037395a3e98bf5339aa14f6f02c002f3ab51"
+	SRC_URI="https://github.com/QKaiser/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
you no need use ${PN} in the static place of the URI
use only ${P}/${PV} where is it necessary

```diff
-SRC_URI="https://github.com/QKaiser/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/QKaiser/cottontail/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```